### DISCUSSION TITLE
Wger as add-on in Home Assistant

### DISCUSSION THREAD

**Discussion by JoeDet** [wger-project/wger#1235] [Ideas]
In this Home Assistant repo, https://github.com/alexbelgium/hassio-addons, Wger has been added as an add-on and its a fantastic solution. Only thing is that it is [not supporting Ingress](https://github.com/alexbelgium/hassio-addons/issues/61#issuecomment-1329513274). This means its only accessible from you local network and not from the gym. Is this something of interest to fix? Would be cool to have Wger fully functional in Home Assistant completely local behind Baby Casas protection.

**Comment by rolandgeider** [comment]
Hi! Glad that you like it!

The helm charts were contributed by the community. What exactly would need to be done? Currently needs to be run on its own domain/subdomain, there are only a handful of places where the URL is hard coded. In any case, feel free to open a new issue in the repo: https://github.com/wger-project/helm-charts/issues

**Reply by JoeDet** [reply]
Hi.

Really like it today and has heaps of potential for improvement. Im no programmer, just above average tech interested, so not sure what would need to be done. Just really liked the idea of having a local gym app and that Home Assistant is becomming a locally hosted one-stop-shop for a lot of my app needs.

**Reply by rolandgeider** [reply]
ah, alright. I'll see what I can do 😄

FYI, there is a [docker-compose](https://github.com/wger-project/docker) setup, perhaps that could be something for you? At least till we find a solution
